cmake/Package.cmake

```diff
@@ -0,0 +1,23 @@
+# -- packaging -----------------------------------------------------------------
+
+set(CPACK_PACKAGE_VENDOR "Tenzir")
+set(CPACK_PACKAGE_CONTACT "engineering@tenzir.com")
+set(CPACK_VERBATIM_VARIABLES ON)
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+set(CPACK_INSTALLED_DIRECTORIES "/var/lib/vast" "/var/log/vast")
+
+set(CPACK_GENERATOR TZST;DEB)
```

Comment:
```suggestion
set(CPACK_GENERATOR "TZST")

if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
  list(APPEND CPACK_GENERATOR "DEB")
endif ()
```

Reply:
The Generator selection is now done from the command line.